IonDotnet/Timestamp.cs

```diff
@@ -361,7 +361,14 @@ private static int GetOffsetMinutes(string s, int startIdx)
                 throw new FormatException(s);
             }
 
-            return hour * 60 + minute;
+            int offset = hour * 60 + minute;
+
+            if (hour > 23 || minute > 59)
+            {
+                throw new FormatException($"{s} - Offset { offset } must be between -1439 and 1439 inclusive and minute between 00 and 59.");
```

Comment:
Since this is on the text parsing path, the user will never actually see the offset in minutes in the input timestamp. In this error message it's probably best to express the bounds in terms of hour and minutes.

Reply:
Done